src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,230 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement saveCarStatement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            saveCarStatement.setLong(1, car.getManufacturer().getId());
+            saveCarStatement.setString(2, car.getModel());
+            saveCarStatement.executeUpdate();
+            ResultSet resultSet = saveCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create "
+                    + car + ". ", throwable);
+        }
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT cars.id AS id, "
+                + "manufacturers.`name` AS manufacturer_name,  "
```

Comment:
```suggestion
                + "manufacturers.name AS manufacturer_name,  "
```

why are there quotes?

Reply:
I wrote queries manually in TablePlus and then copied them into DAO.
Fixed.